zombienet_tests/async_backing/async-backing-test.toml

```diff
@@ -0,0 +1,32 @@
+[settings]
+timeout = 1000
+
+[relaychain]
+default_image = "{{ZOMBIENET_INTEGRATION_TEST_IMAGE}}"
+chain = "rococo-local"
+default_command = "polkadot"
+
+  [relaychain.genesis.runtime.runtime_genesis_config.configuration.config]
+  max_validators_per_core = 2
+
+  [relaychain.default_resources]
+  limits = { memory = "4G", cpu = "2" }
+  requests = { memory = "2G", cpu = "1" }
+
+  [[relaychain.nodes]]
+  name = "alice"
+  args = [ "-lparachain=debug"]
+
+  [[relaychain.nodes]]
+  name = "bob"
+  image = "{{ZOMBIENET_INTEGRATION_TEST_SECONDARY_IMAGE}}"
+  args = [ "-lparachain=debug"]
+
+[[parachains]]
+id = 100
+cumulus_based = true
+
+  [parachains.collator]
+  name = "{{COL_IMAGE}}"
+  command = "undying-collator"
```

Comment:
this is not cumulus based collator (implied by `cumulus_based = true`), let's use a cumulus collator for this test.

Reply:
So the CI Collator image is not Cumulus based, and hence we shouldn't use this? I can see that the other zombienet CI tests do not specify `cumulus_based = true`.

Reply:
We can use either Cumulus or test parachain collators. For this test I suggest we use a cumulus based collator.